dev-python/authlib/authlib-1.5.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10,11,12,13} )
+
+DISTUTILS_USE_PEP517="setuptools"
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A Python library in building OAuth and OpenID Connect servers and clients."
+
+HOMEPAGE="https://authlib.org/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="django flask jose test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/cryptography[${PYTHON_USEDEP}]
+	django? (
+		dev-python/django[${PYTHON_USEDEP}]
+	)
+	flask? (
+		dev-python/flask[${PYTHON_USEDEP}]
+		dev-python/flask-sqlalchemy[${PYTHON_USEDEP}]
+	)
+	jose? (
+		>=dev-python/pycryptodome-3.10[${PYTHON_USEDEP}]
+		<dev-python/pycryptodome-4[${PYTHON_USEDEP}]
+	)
+"
+BDEPEND="
+	test? (
```

Comment:
Don't you need the optional RDEPs for tests?

Reply:
Do you mean I should include RDEPEND in BDEPEND?

Reply:
No, I mean that if these optional dependencies are required during testing, then you need to inline them all into `test?` because you can't expect people to enable all USE flags to test stuff.

Reply:
All I'm saying is make sure that tests pass with USE=-* and all optional dependencies uninstalled.

Reply:
I verify that `ebuild pipenv-2025.0.2.built test` works without safety installed on the system. Also, I found out that two test dependencies that were declared are no longer needed. Is there any other change required in the ebuild?